static/app/components/searchQueryBuilder/tokens/filter/valueCombobox.tsx

```diff
@@ -174,30 +175,6 @@ function getValueAtCursorPosition(text: string, cursorPosition: number | null) {
 
   return '';
 }
-/**
- * Replaces the focused filter value (at cursorPosition) with the new value.
- *
- * Example:
- * replaceValueAtPosition('foo,bar,baz', 5, 'new') => 'foo,new,baz'
- */
-function replaceValueAtPosition(
```

Comment:
OOPS now i feel bad since this function was just moved and I left like 3 comments, still though!